setup.cfg

```diff
@@ -23,6 +23,7 @@ classifiers =
     Programming Language :: Python :: 3.9
     Topic :: Software Development :: Libraries :: Python Modules
     Topic :: Scientific/Engineering :: GIS
+    Typing :: Typed
```

Comment:
So this tells mypy that types are present?

Reply:
https://mypy.readthedocs.io/en/stable/installed_packages.html

> If you would like to publish a library package to a package repository yourself (e.g. on PyPI) for either internal or external use in type checking, packages that supply type information via type comments or annotations in the code should put a py.typed file in their package directory.